Index: ../resharper/Plugin.props
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><Project>\r\n\r\n    <PropertyGroup>\r\n\r\n        <SdkVersion>2024.3</SdkVersion>\r\n\r\n        <Title>Abp Insight</Title>\r\n        <Description>Adds navigation, auto-completion and inspections specific to the Abp Framework.</Description>\r\n\r\n        <Authors>Carbon</Authors>\r\n        <Copyright>Copyright $([System.DateTime]::Now.Year) Maintainers of AbpInsight</Copyright>\r\n        <PackageTags>resharper plugin</PackageTags>\r\n        <PackageProjectUrl></PackageProjectUrl>\r\n        <PackageLicenseUrl></PackageLicenseUrl>\r\n        <PackageIconUrl></PackageIconUrl>\r\n    </PropertyGroup>\r\n\r\n</Project>\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../resharper/Plugin.props b/../resharper/Plugin.props
--- a/../resharper/Plugin.props	(revision 645b8e07c0d835032f9f48d5d08a40344b03b968)
+++ b/../resharper/Plugin.props	(date 1732003021750)
@@ -1,18 +1,9 @@
 <Project>
 
     <PropertyGroup>
-
         <SdkVersion>2024.3</SdkVersion>
-
         <Title>Abp Insight</Title>
-        <Description>Adds navigation, auto-completion and inspections specific to the Abp Framework.</Description>
-
         <Authors>Carbon</Authors>
-        <Copyright>Copyright $([System.DateTime]::Now.Year) Maintainers of AbpInsight</Copyright>
-        <PackageTags>resharper plugin</PackageTags>
-        <PackageProjectUrl></PackageProjectUrl>
-        <PackageLicenseUrl></PackageLicenseUrl>
-        <PackageIconUrl></PackageIconUrl>
     </PropertyGroup>
 
 </Project>
Index: src/main/resources/META-INF/plugin.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><idea-plugin require-restart=\"true\">\r\n    <id>carbon.abpInsight</id>\r\n    <name>Abp Insight</name>\r\n    <version>2024.3.0</version>\r\n    <vendor>Carbon</vendor>\r\n    <idea-version since-build=\"2024.3\"/>\r\n    <depends>com.intellij.modules.rider</depends>\r\n\r\n    <description><![CDATA[\r\n<p>Adds navigation, auto-completion and inspections specific to the <a href=\"https://abp.io/\">Abp Framework</a>.</p>\r\n<p>\r\n<strong>Features</strong>\r\n<ul>\r\n  <li>Analyze Abp module dependencies and potential problems.</li>\r\n  <li>And much more!</li>\r\n</ul>\r\n</p>\r\n]]>\r\n    </description>\r\n\r\n</idea-plugin>\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/resources/META-INF/plugin.xml b/src/main/resources/META-INF/plugin.xml
--- a/src/main/resources/META-INF/plugin.xml	(revision 645b8e07c0d835032f9f48d5d08a40344b03b968)
+++ b/src/main/resources/META-INF/plugin.xml	(date 1732003444704)
@@ -1,9 +1,7 @@
 <idea-plugin require-restart="true">
-    <id>carbon.abpInsight</id>
+    <id>Carbon.AbpInsight</id>
     <name>Abp Insight</name>
-    <version>2024.3.0</version>
     <vendor>Carbon</vendor>
-    <idea-version since-build="2024.3"/>
     <depends>com.intellij.modules.rider</depends>
 
     <description><![CDATA[
